api.js

```diff
@@ -131,8 +132,8 @@ apiRouter.get('/gltf/:tid', async (req, res) => {
  */
 apiRouter.post('/event', (req, res) => {
     if (req.body.event === 'onshape.model.translation.complete') {
-        // Save in Redis so we can return to client later (& unregister the webhook).
-        redisClient.set(req.body.translationId, req.body.webhookId);
+        // Save in memory so we can return to client later (& unregister the webhook).
+        inMemoryDataStore[req.body.translationId] = req.body.webhookId;
```

Comment:
We could clear out the `inMemoryDataStore[req.body.translationId]` once we unregister the webhool